verify/cockpit-verify

```diff
@@ -28,7 +26,17 @@ while true; do
     git -C cockpit fetch origin master
     git -C cockpit reset --hard FETCH_HEAD
     cockpit/test/vm-download --prune
+
+    if ! git diff -C cockpit --exit-code origin/master package.json; then
+        install=1
+    fi
+
+    if [ "$install" = "1" ]; then
+        (cd cockpit && npm install)
```

Comment:
During the first run, this will do npm install from the first branch it's on, but in subsequent runs it compares the package.json to master. Essentially something different could be compared than is executed.
Maybe store the revision to compare against? Or a checksum?